### PR TITLE
[Webportal] Support Job Priority in job-list

### PR DESCRIPTION
**Pull request by yiyione**
Support UX of Job Priority #5417

### CODE REVIEW

File: src/webportal/src/app/job/job-view/fabric/JobList/Filter.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import { TestSpan } from "@azure/core-tracing";
```
Comment (debuggy) [inline src/webportal/src/app/job/job-view/fabric/JobList/Filter.js:1]: maybe should be deleted

File: src/webportal/src/app/job/job-view/fabric/JobList/Filter.js [hunk 2]
```diff
@@ -40,6 +45,9 @@ class Filter {
       if (Array.isArray(users)) {
         this.users = new Set(users);
       }
+      if (Array.isArray(priorities)) {
```
Comment (debuggy) [inline src/webportal/src/app/job/job-view/fabric/JobList/Filter.js:46]: maybe this.priorities?

File: src/webportal/src/app/job/job-view/fabric/JobList/Filter.js [hunk 3]
```diff
@@ -62,6 +70,16 @@ class Filter {
     if (users && users.size > 0) {
       query.username = Array.from(users).join(',');
     }
+    if (priorities && priorities.size > 0) {
+      query.jobPriority = Array.from(priorities).map(priority => {
+        switch (priority) {
+          case 'Opportunistic': return 'oppo';
+          case 'Product': return 'prod';
```
Comment (debuggy) [inline src/webportal/src/app/job/job-view/fabric/JobList/Filter.js]: there is some lint error could be fixed by prettier

File: src/webportal/src/app/job/job-view/fabric/JobList/Filter.js (hunk 1 above)
Comment (yiyione) [reply src/webportal/src/app/job/job-view/fabric/JobList/Filter.js]: Done.

File: src/webportal/src/app/job/job-view/fabric/JobList/Filter.js (hunk 2 above)
Comment (yiyione) [reply src/webportal/src/app/job/job-view/fabric/JobList/Filter.js:52]: Done.

File: src/webportal/src/app/job/job-view/fabric/JobList/Filter.js (hunk 3 above)
Comment (yiyione) [reply src/webportal/src/app/job/job-view/fabric/JobList/Filter.js]: Done.

**Review by debuggy — APPROVED** [review]

**Pull request merged**